### ISSUE TITLE
Fix tooltips to better represent design requirements

### ISSUE THREAD

**Issue by turt2live** [vector-im/element-web#11486]
We need them to go up, down, left, and right but are currently stuck with only right. This can at best look weird and at worst prevent users from interacting with the component.

While we're here, we should fix tooltips in general:
* https://github.com/vector-im/riot-web/issues/10155
* https://github.com/vector-im/riot-web/issues/9946
* https://github.com/vector-im/riot-web/issues/13898
* Each component that uses them is required to do math to position them - should be magic (from the component's POV)

**Comment by turt2live** [comment]
we did this

**Comment by t3chguy** [comment]
Nope, they still don't go in the right directions matching the designs

**Comment by ShadowJonathan** [comment]
https://github.com/vector-im/element-web/issues/18114 mentions that the SSO popups are too quick and too close to the cursor, FWIW

**Comment by t3chguy** [comment]
This happened via Compound
